packages/manager/apps/container/src/container/mfa-enrollment/MfaEnrollment.tsx

```diff
@@ -6,20 +6,47 @@ import img from '@/assets/images/mfa/double-authentication.png';
 import img2x from '@/assets/images/mfa/double-authentication@2x.png';
 import img3x from '@/assets/images/mfa/double-authentication@3x.png';
 
-import style from './style.module.scss';
+import { OsdsButton, OsdsText, OsdsIcon } from '@ovhcloud/ods-components/react';
+import {
+  ODS_BUTTON_SIZE,
+  ODS_BUTTON_VARIANT,
+  ODS_TEXT_COLOR_HUE,
+  ODS_TEXT_LEVEL,
+  ODS_TEXT_SIZE,
+  ODS_ICON_NAME,
+  ODS_ICON_SIZE,
+} from '@ovhcloud/ods-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { Subtitle } from '@ovhcloud/manager-components';
 
 type MfaEnrollmentProps = {
-  forced?: boolean;
-  onHide: CallableFunction;
+  readonly forced?: boolean;
+  readonly onHide: CallableFunction;
```

Comment:
Try declaring the component as a `const` then, just like you did with the sub components (and it will be more consistent in the same file).
And don't resolve the comments when it's a question please.